TurtleShroom Productions VIII, Part II: Eclectic Boogaloo

TurtleShroom opened the pull request:
Below is the letter I wrote to Loup on the Discord chat room.

---------------------------------------------------

_Good morning. I expected you to do this._

_That is insulting, to say the least, so I'll revisit what I posted in the PR._

_Take your consensus, place it upon a cactus, and go sit upon that cactus. I don't care what you think, I don't value what you think, and I don't like you. I don't want to ever have consensus with you, because you have never cared about what I think or considered that my code may still be good despite you not liking it._

_Last night, Tanner and I challenged your dictatorship over this Mod. My code was good, my code was worthy, and my code belonged in the Mod. I told you I would get it Committed, and Commit it I did.
The only mistake is you reverting it, and I want it back._

_Let me make this very clear to you, Loup._

_I don't like you. I despise your opinions. I despise your obstructionism, and I despise your reversion of all my hard work. Tanner saw the truth and knew this code was good for the Mod. He did what was best for the Mod. You only do what's best for yourself. You want to make sure the Mod is yours and is exactly the way YOU want it, and to heck with everyone else._

_You're adding stuff from my code that you like and giving me the shaft elsewhere. You're making it into your code, which matches your view. You never cared about me or my contributions to this Mod. You only cared about making my code match what you want this Mod to be._

_If you're so into democracy and "concertation", you should consider that Tanner and I make two in favor, while you are only one against. Add Maelgwyn (who also has just problems with you) to that and you get three. You've been overruled. I've had to take getting overruled and halted for months on end. It's about time you taste what that feels like as well. The fact that you are committing my work now that I got your attention shows how little you think of my code._

_The PR is reopened. I didn't put months into this code to have it cast aside because it challenges your dominance of this Mod._

_-TURTLESHROOM_

loup99 commented:
@TurtleShroom 
I know there is a lot of passion in this open letter and a lot which you were unhappy with that didn't end up as you wanted, such as me saying no to your prestige requirement changes for Erik W's special titular titles, or when we discussed the investitures and our conclusion ended up being that the system wasn't fit as it was designed for the XIth century. Yet, I would nonetheless like you to take a few moments and actually read through this response that I'm posting here as the discussion no longer is of a private nature. 

Sometimes it can be good to just get out your anger when you are upset over something or doesn't like a person. In many cases it may not be done, but here I think you really needed it, and I'm not going to take any of it personally, because it doesn't matter in the grand schemes of things. I'm not angry, I'm not worked up or upset in any way that you decided to share your thoughts openly. 

The content in your previous pull request which hasn't been merged into the mod was subject to an ongoing review and discussion around it. At some point it either became too much for you to continue the discussion and debate, or you were at the end of your tether or you for some other reason decided to find other channels to get the merge which you wanted to get. It isn't problematic that you try this through different methods, on the countrary, insight from multiple persons is often valuable.

Remember however that this is a mod. The code we add here to share with other developers needs to respect a few barebone standards:

1. It needs to work. This may seem obvious, but you can never be enough clear with it. Yours did not live up to this, there were mistakes and bugs. I have already explained this to ts and I believe I told you that by rejecting the review, but I will reiterate it here too for the sake of clarity. Most of the code issues had been caused by bad merges, for your reference, just in case you want to keep this in mind if you do this again.
2. It needs to have been tested. This one ties into the former, but some of the issues which had crept in to your mod which CTDs the game (<<<<<<< HEAD for instance) indicates that the branch hadn't been properly tested. Maybe it had, but then it still needed a review to spot those problems.
3. It needs to fit into the mod. Here it is entirely subjective, and all developers may have a different opinion, which we all need to take into account and listen to when finding common solutions. 

Adding to these points, I would like to clarify the priorities from which I have been working on this project:

1. Finalise the secret societies, fix bugs, implement requests and release a new version which is up to date with 2.7.1. 
2. Do everything needed in order to finalise 1.0.0, for which the priorities are listed in the sticky development thread.

Note that this is how I have personally been prioritising things for the work I do personally, and that it isn't a rule or guideline for anyone else,as the most important is to develop the mod and that everyone like what they are doing.

Concerning the pull-request, what I decided to do this morning when I realised there had been a mistake with integration in that the code wasn't ready for implementation, was to go through the files one by one to see what worked and could be improved, given that I realised that you weren't interested in working through the pull-request format anymore. Hence the notes on the fact that the process I was about to engage in would take a while, as I had  to go through commit by commit due to the pull request having been closed. 

Hopefully we will from this point and onwards find solutions and agreements, so that we can continue to improve the mod in concertation and collaboration. If you would prefer me to work in another way to progressively implement the code, please tell me here, I'm obviously all ears as it you who created the pul-request in the first place. 

As a more private aside, which I hope you will pardon me for including here, I have a few citations too in store, which I thought might interest you as the one on the steppe culture. Didn't post them on the Discord yesterday because you were a bit angered and it didn't seem like it was the right time to do so. 

Thank you for reading,
Loup

TurtleShroom commented:
I removed the prestige requirements and didn't add back anything we had already resolved.

1. I tested the code for _**decades**_ in-game, and it works just fine. I experienced no frequent crashes or glitches that would indicate a collapse. I have tested this code _and Tanner has tested this code_, and it meets his standards as well as mine. I'm playing a game on my code right now, and it works perfectly, as intended. Don't lie and tell me it's busted: Tanner and I have tested it and we know it isn't.

2. To my knowledge, there are no ">>>>>HEAD REFS<///" corrupted code in the PR, leftover from merges. I removed those. If there was, it'd have crashed. It didn't.

3. Tanner and I know that it does fit into the Mod, so screw you.

4. I am interested in working with the PR  system. I reopened my PR right here, after all.

------------------------------------------------------------------------------------------------

Show a little democracy and accept the changes. Tanner and I outnumber you. Now, it's time for _you_ to sing in "concertation" with us. I can probably get Mael to back me on this if you need three people to override your dictatorship.

That pathetic excuse of an "ongoing review" was intentionally dragged on for months with the express motive of delaying my PR as long as possible, in order that I would be exhausted and submit to your interpretation of the code.

The review was _**finished**_. Tanner completed it when you wouldn't. Tanner fully agrees with me that my PR was ready. Just because you disagreed with me on something didn't mean it was wrong.

You want this to be your Mod, me and Tanner be darned, and with Erik and Enlil gone, no one can challenge your dictatorship. You want all code to match exactly what YOU want it to be, with no concern or interest in others.

So once again I say, screw you; go sit on a cactus.

loup99 commented:
1. I have talked with ts about the issues in the pull request and he agreed with and apologised for what happened. That isn't the question though, because I don't find it problematic that he missed some code or wasn't invested in this. I'm not going to make any further remarks on that, because it is honestly completely off-topic what has been discussed privately. 
2. There are, see my review. Or actually it doesn't matter anymore, because in that particular file I sorted it out in commit 8cf198f from this morning.
3. Okay, then I will send you a message on Discord to continue the discussion where it left off.

loup99 commented:
Review of the "new" additions:

> Turtleshroom's suggestion: just characters less likely to force Intolerance unless other traits or natures are present and just characters of Sectarian Religions are no longer less likely to enact intolerance

Again, I told you I disagreed on this change and justified it with historical examples. I think it should stay as it is at this point. 

> The two ROMAN EMPIRES do not use the ROMAN Imperial Administration

Note that the law is redundant: they already have those benefits.

> Suggestion to move date 10 years later so Augustus is not portrayed as child

The best is to keep the current date. Yes, Augustus portrayed as a child, but for that we had another solution in mind (special graphical culture portraits).

loup99 commented on WTWSMS/localisation/MZ.csv:
```diff
@@ -183,8 +183,13 @@ fund_shahnameh_addition_desc;Fund an addition to the Shahnameh to record the tal
 yazidi_DESC;Yazidis believe that God created the world and left seven angels in charge, the chief of whom is the Peacock Angel, Malek Taus, often mistakenly confused with the Abrahamic faith's Satan. The Yazidi religion is mainly held by ethnic Kurds, living in Zagros mountains. The religion is syncretic and include elements from old Mesopotamian religions, Zoroastrianism, Christianity, and other ...
 priscillian_carnivore;Carnivourous Clergy;;;;;;;;;;;;;x
 priscillian_carnivre_desc;This Priscillianist priest, monk, or nun eats meat, in direct violation of the traditions and ascetic, ecclesiastical requirements of the Priscillian sect. To do this is considered a grave sin and makes the cleric unworthy of their position.;;;;;;;;;;;;;x
+<<<<<<< HEAD
```
Bad merge.

loup99 commented on WTWSMS/localisation/MZ.csv:
```diff
@@ -183,8 +183,13 @@ fund_shahnameh_addition_desc;Fund an addition to the Shahnameh to record the tal
+=======
+donamonaprisc_decadent;Defier of Asceticism;;;;;;;;;;;;;x
+donamonaprisc_decadent_desc;This character is decadent and lives in excess and/or in counter to their faith's paramount virtues of asceticism. He or she makes a mockery of their religion and has earned the ire and disgust of all virtuous members of his or her faith.;;;;;;;;;;;;;x
+>>>>>>> parent of da3dcfe6... Integrating @TurtleShroom 's Changes #224
```
Broken code.

loup99 commented on WTWSMS/localisation/MZ.csv:
```diff
@@ -183,8 +183,13 @@ fund_shahnameh_addition_desc;Fund an addition to the Shahnameh to record the tal
+<<<<<<< HEAD
 defiance_of_asceticism;Defier of Asceticism;;;;;;;;;;;;;x
 defiance_of_asceticism_desc;This character is decadent and lives in excess and/or in counter to their faith's paramount virtues of asceticism. He or she makes a mockery of their religion and has earned the ire and disgust of all virtuous members of his or her faith.;;;;;;;;;;;;;x
+=======
```
Unsolved issue.

loup99 commented on WTWSMS/localisation/VIET_religion_jewish.csv:
```diff
@@ -34,8 +34,8 @@ israel.108.a;Welcome! We need all the bright minds we can get.;;;;;;;;;;;x
+viet_opinion_dismiss_minority;Did not accept Aramean comparisson to Jews;;;;;;;;;;;x
 viet_opinion_accepted_arameans_first;Extended protection to the Arameans;;;;;;;;;;;x
 viet_opinion_refused_arameans;Concentrated in protecting the Jewish people;;;;;;;;;;;x
 viet_opinion_accepted_arameans_again;Re-affirmed protection of Arameans;;;;;;;;;;;x
```
You have added typoes to this file.

loup99 commented on WTWSMS/localisation/WtWSMS_misc.csv:
```diff
@@ -186,4 +186,14 @@ danced_with_the_dead;Danced with the Dead;Joue avec la mort;;;;;;;;;;;;x
 greek_official_language;Greek official language;;;;;;;;;;;;;x
-greek_official_language_2;Greek official language;;;;;;;;;;;;;x
\ No newline at end of file
+<<<<<<< HEAD
```
Bad merge.

loup99 commented on WTWSMS/localisation/WtWSMS_misc.csv:
```diff
@@ -186,4 +186,14 @@ danced_with_the_dead;Danced with the Dead;Joue avec la mort;;;;;;;;;;;;x
\ No newline at end of file
+<<<<<<< HEAD
+greek_official_language_2;Greek official language;;;;;;;;;;;;;x
+=======
```
This needs to be resolved.

loup99 commented on WTWSMS/localisation/WtWSMS_misc.csv:
```diff
@@ -186,4 +186,14 @@ danced_with_the_dead;Danced with the Dead;Joue avec la mort;;;;;;;;;;;;x
+donatist_redeemed_desc;This character was once a traditor to his Donatist faith, but like the prodigal son of old, he has come back into his faith, where he has been welcomed with open arms.;;;;;;;;;;x
+priscillian_carnivore;Carnivore;;;;;;;;;;x
+priscillian_vegetarian;Vegetarian;;;;;;;;;;x
+>>>>>>> parent of da3dcfe6... Integrating @TurtleShroom 's Changes #224
```
Again, broken.

loup99 commented on WTWSMS/localisation/MZ.csv:
```diff
@@ -166,6 +166,9 @@ become_a_vegetarian_desc;Become a vegetarian to show your devotion to your faith
 fund_shahnameh_addition;Fund Addition to the Shahnameh;;;;;;;;;;;;;;x
+mandaeism;Mandaeism;;;;;;;;;;;;;x
+mazdaki_DESC;Mazdak was a Zoroastrian priest;;;;;;;;;;;;;x
+mandaeism_DESC;This dualistic faith;;;;;;;;;;;;;x
```
Truncated code which is already present in vanilla, which I removed.

loup99 commented on WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt:
```diff
@@ -915,6 +878,101 @@ secret_taoism_community = {
+	same_religion_opinion = 10
+
+}
+
```
This is a duplicate with the above that you removed.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
+		capital = 198 # Avila
+		
+		title = "PRISCMASTER"
+		foa = "HOCHMEISTER_FOA"
```
Why this foa?

TurtleShroom replied:
I couldn't think of another, but it makes sense.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
+}
+	
+d_sethian_holy_order = {
+		color = { 0.123 0.123 0.123 }
```
Lacks a colour two.

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
+d_sethian_holy_order = {
+		color = { 0.123 0.123 0.123 }
+		
+		graphical_culture = hashshashingfx # Mystic warriors of the Middle East
```
That gfx doesn't fit for a Christian holy order.

TurtleShroom replied:
The Sethians are not actual Christians, even though they are syncretized with Christianity. They are polytheistic and may predate Christianity itself. Most importantly, Sethians were mystical and of Middle Eastern origin. Not only does it look cool, it makes more sense than the usual Christian symbols of crosses and the like. Sethians used snakes as their symbol, remember.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
+		
+		mercenary_type = sacred_band
+		
+		strength_growth_per_century = 0.05
```
This means that they will decrease with time.

TurtleShroom replied:
A reading of one means it doubles (one hundred percent) each century, which jacks the cost of until it's unaffordable.

This is strength GROWTH, which means it grows by five percent.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
+		
+		name_tier = DUKE
+		
+		monthly_income = -0.5 # Asceticism (this is supposed to reduce the income by half)
```
Yeah, but this leads to them being in debt.

TurtleShroom replied:
No, it halves their income. It doesn't cost them money.

loup99 replied:
Are you sure on that? Because that negative one would be applied regardless of whether or not they gain money.

loup99 replied:
Reading up on it, it must be an integer between 1 and 512.

TurtleShroom replied:
Ah, I see the problem. The negative number isn't recognized.

loup99 replied:
Yes, that is neither an integer nor between 1 and 512.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
 
+}
+
+c_priscillianism_holy_order = {
```
Should be duchy-level.

TurtleShroom replied:
That was intentional, so that the Priscillian Pope could vassalize them, or the player could transfer them to the Priscillian Pope. The Priscillian Pope should be King-tier, but I stopped disputing that a long time ago.

loup99 replied:
This applies to all holy orders though. To my knowledge this isn't an exception.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt:
```diff
@@ -286,4 +286,65 @@ d_wasilla_guardians = {
+		
+		male_names = { Priscillian Marcus Helpidus }
+		
+		strength_growth_per_century = 0.15
```
This will make them far too weak.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -333,7 +333,7 @@ k_venice = {
 	}
 	
-	title = "DOGE"
+	title = "DOGE_WOW"
```
What happened here?

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -2609,7 +2612,9 @@ e_moors = {
 	# Creation/Usurp Trigger
 	allow = {
 		OR = {
+			culture = maghreb_arabic
```
This is an Arabic culture, so it doesn't fir for e_moors.

TurtleShroom replied:
The Maghrebs occupied the area of the Moors and northern Africa. Maghrebs are not your normal Arabs. They are part of that region.

loup99 replied:
This isn't a geographic entity though. It is cultural for the "Moors".

TurtleShroom replied:
[Maghrebis _**ARE**_ Moors.](https://www.google.com/search?hl=en&q=are+maghrebis+moors&sourceid=navclient-ff&rlz=1B3GGGL_en___US363&ie=UTF-8&gws_rd=ssl)

loup99 replied:
Inhabitants of the geographic region of Maghreb, yes. That wasn't what I referred to though, neither what the title represents.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt:
```diff
@@ -2609,7 +2612,9 @@ e_moors = {
 		OR = {
+			culture = maghreb_arabic
 			culture = berber
+			culture = berger_germanic
```
Looking at the name list, I'm not sure this fits.

TurtleShroom replied:
Crap. That was a typo. I will fix it immediately.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt:
```diff
@@ -678,3 +678,36 @@ d_vasconic_pagan_reformed = {
+	primary = yes
+	
+	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
+}
```
This title already exists in landed_titles.txt.

TurtleShroom replied:
I was certain I moved that back.

TurtleShroom replied:
I did move it back. It's not in "LANDED_TITLES".

loup99 replied:
You didn't move it back, you moved it to another file. In the master it is in landed_titles on line 40876.

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_zusk_titles.txt:
```diff
@@ -110,7 +110,8 @@ e_greece = {
 	#Lots of flavor names, for fun :3
-	  
+	
+	tsakonian = Sparta
```
The Tsakonian culture isn't Spartan in that sense, so the name does not fit.

TurtleShroom replied:
Okay.

loup99 commented on WTWSMS/common/landed_titles/WtWSMS_zusk_titles.txt:
```diff
@@ -302,16 +303,11 @@ e_khitan = {
-		Hendeji Houzhu Huaizong Huanzong Huizong Hula Jingzong Kangzong Liezu Lizong Mingzong Muzong Ningzong Qinzong Renzong Ruizong 
-		Shangdi Shaozhu Shenzong Shizong Shizu Shunzong Su Suzong Taizong Tianzuo Wa Wenzong Wuwang Wuzong Xiangzong Xianzhi Xianzhu Xianzong Xiaozong 
-		Xingzong Xizong Xuanzong Yelan Yingzong Yizong Yuanzong Zhangzong Zhaogu Zhaozong Zhengzong Zhenzong Zhongzhu Zhongzong Zhuangzong Zuwu
-	}
```
What happened here?

TurtleShroom replied:
I added the Khitan names back.

loup99 replied:
What is the point of the removal then? I don't get the nature of this change, so I'm always wary of encoding issues.

TurtleShroom replied:
I didn't remove the names. I reordered them: instead of being on multiple lines, they are now continuous, as they should be.

loup99 replied:
They are meant to be multiple lines. Check the name lists in the cultures files.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
```
While Cyprus was in its nature of a Greek connection at this time, I can see from where you are coming. Yet I do not think this change makes much sense given that the title is titular and requires the ruler to be independent, which means that it makes sense to have Greek for a Greek cultured-area.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			}
 			is_adult = yes
-			prestige = 5000
+			prestige = 2000
```
This prestige rebalancing you did unilaterally has already been rejected at least three times.

TurtleShroom replied:
I was certain I changed that one back. I apologize.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
-		hellenic_pagan = 600
-		bosporan_faith = 200
+		hellenic_pagan = 200
+		bosporan_faith = 600
```
Why the reshuffle?

TurtleShroom replied:
Tauricia was sacrosanct to the Hypsistarians and had little interest from the Hellenics. The Hypsistarian religion began in Tauricia, and Tauricia was largely Hypsistarian country. There shouldn't be a Hellenic interest in this title, TBH.

loup99 replied:
I'm not opposed to raising the Hypsistarian faith, but I disagree on your other change. Tauricia was an important part of the Hellenic world and of religious importance, justifying this value which comes from ACR.

TurtleShroom replied:
How was Tauricia, in any way, _religiously_ important to the Hellenic faith? I know why it's important to Rome in secular terms, obviously, but not to the Hellenic _religion_.

loup99 replied:
Do you know the relation between the secular and religious in the Hellenic culture and faith?

TurtleShroom replied:
No, but I'll take your word for it and return it.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
+					OR = { 
+						culture_group = byzantine
+						culture_group = latin
+						has_romance_culture_trigger = yes
```
I have already explained why ai = no stays, and I don't get why you added the Romance group.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -15805,6 +15825,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```
See previous review.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -16046,11 +16067,9 @@ e_armenia = {
-				}
+			OR = {
+				culture = armenian
+				religion_group = christian
```
A detailed argumentation on this is available in the previous review.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -30620,6 +30640,8 @@ e_italy = {
 			is_republic = yes
 		}
+
+		title = DOGE_WOW
```
Why?

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -40383,918 +40424,371 @@ e_latin_empire = {
 	}
 }
 
-d_miaphysite = {
```
From this line and onwards the file is completely messed up, making any review impossible. Discard your code and start over from scratch.

TurtleShroom replied:
I moved it back to where I had removed it, in accordance with _**your**_ orders. It worked fine when I had it in its own file.

TurtleShroom replied:
How is it messed up?

![new bitmap image](https://user-images.githubusercontent.com/17663470/28176753-87572792-67c6-11e7-9877-c14617d6fcef.png)

It works perfectly on my end.

loup99 replied:
I'm surprised GitHub works on that old OS, to be honest. 

On topic, I think it is caused by you moving out some other orders. The problem is that it is impossible to review as I can't see what you changed or not.

TurtleShroom replied:
That's a Windows Seven OS.

loup99 replied:
The one which has been discontinued? But I thought it was older: must be because of the old graphics you use.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -40346,19 +40368,38 @@ e_latin_empire = {
+			culture = greek
+			culture = roman
+			religion = orthodox
+			religion = iconoclast
```
Huh?

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -40346,19 +40368,38 @@ e_latin_empire = {
 	allow = {
 		OR = {
+			has_romance_culture_trigger = yes
+			culture_group = latin
```
Disagreed, there is no reason to tie this title to these cultures.

TurtleShroom replied:
_**LATIN**_ Empire.

loup99 replied:
Latin as in Latin crusaders for the Pope in Rome.

loup99 commented on WTWSMS/common/landed_titles/landed_titles.txt:
```diff
@@ -40346,19 +40368,38 @@ e_latin_empire = {
+					character = ROOT
+				}
+		}
+
```
First of all this condition isn't necessary, secondly remember that this is the Latin Empire.

TurtleShroom replied:
In the file Latin Empire's circumstances are as follows:

1. You  must control the Latin Empire's historic lands (Thrake, Kaliopolis, and Constantinople)
2. You must not be Orthodox or Iconoclast (post-Schism)
3. You must not hold the Byzantine Empire
4. Rome has not been restored

These are near identical to the Vanilla rendition. Besides, the Latins have always wanted to hold the City of the World's Desire.

loup99 replied:
1. Disagreed, that is far too much railroading.
2. Not really relevant at this point as the schism doesn't exist ingame.
3. Obviously, yes.
4. Debatable.

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -390,8 +390,6 @@ d_sky_lords = {
 	# Parent Religion 
```
This file is not ready for implementation for a series of reasons:

1. These changes need a detailed explanation in a dedicated issue with explanations on why this was needed and what the aim with the changes were. 
2. Some of the changes need some tinkering on balance.
3. There has been a lot of reshuffling, which we do in separate files to avoid compatibility problems.

loup99 commented on WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt:
```diff
@@ -158,6 +159,7 @@ title_court_celtic_poet = { # Celtic Poet/Bard
 	}
 	
 	gain_effect = {
+		add_trait = poet
```
This serves no purpose.

TurtleShroom replied:
Fair enough.

loup99 commented on WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt:
```diff
@@ -132,20 +132,21 @@ title_sigecempa = {
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
```
Why?

TurtleShroom replied:
The prestige level was too low. Bards were a VERY big deal in that societiy. Like the Griot, they should have far higher prestige and salary rates because of it. Both of those titles' prestige and salary are too low.

loup99 replied:
Disagreed, it is a honorific title which would only be used intermittently in the case of the court poets.

loup99 commented on WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt:
```diff
@@ -132,20 +132,21 @@ title_sigecempa = {
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
```
Why?

TurtleShroom replied:
One quarter of one dignity doesn't  show up in the game. Dignity represents how they are treated in terms of how titles are selected as primary.

loup99 replied:
It doesn't need to show up though, as far as I know it is still calculated in the background.

IhateTrains commented:
@TurtleShroom 
Sorry for offtopic, but could you give me your Discord tag?

loup99 commented:
@IhateTrains His tag is #7514 What is yours by the way?

tannerflick4 commented:
Hey @loup99 can you invite me to your discord server for the mod?

IhateTrains commented:
My tag is #5378

loup99 commented:
@tannerflick4 I don't have any. Only a PM with TurtleShroom.

tannerflick4 commented:
Ok. No problem. Also, if you could switch the court_poet stuff for me, that'd be great because my git desktop is not working at all today,

loup99 commented:
@tannerflick4 : Here you go, I created one for the occasion: https://discord.gg/W7Rc3P Always handy to have.

TurtleShroom commented on WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt:
```diff
@@ -915,6 +878,101 @@ secret_taoism_community = {
 	}
 }
 
+<<<<<<< HEAD
```
You are wrong. **_Yesterday_**, I had to synchronize with your new updates to the Master line of the Mod. When Tanner Committed the code, we had both already tested it in game. This mistake is _brand new_.

You are absolutely wrong.

loup99 replied:
Are you joking with me? I linked you to this yesterday, and reminded you of it before the old pull was closed https://github.com/loup99/WtWSMS/commit/0adebcde62546134b1bb2faad62016ca2fdd8054#diff-b7f0f652709d2e453cc5b687c40fccedR881

loup99 replied:
Here is my 5-day old post initially notifying you of this, which has stayed throughout the pull-requests: https://github.com/loup99/WtWSMS/pull/224#discussion_r126287245

TurtleShroom commented on WTWSMS/common/event_modifiers/MZ_event_modifiers.txt:
```diff
@@ -1,8 +1,15 @@
+#														#
+# STRATEGIC EFFECTS - A.K.A. "TRIGGERED MODIFIERS"      #
+#														#
+#########################################################
```
I preserved that when I converted it from Scripted Triggers to Event Modifiers.

loup99 replied:
Doesn't concern the whole file, so it makes no sense.
